plugin/auto/setup_test.go

```diff
@@ -126,3 +126,44 @@ func TestAutoParse(t *testing.T) {
 		}
 	}
 }
+
+func TestSetupReload(t *testing.T) {
+	tests := []struct {
+		name    string
+		config  string
+		wantErr bool
+	}{
+		{
+			name: "Blacklist 1",
```

Comment:
These are odd names for the test ... I'm not getting how "Blacklist" relates to the test performed.  How about "reload".

Reply:
Thanks a lot! I fixed it now.